packages/plugin-blockquote/src/BlockquoteDomParser.ts

```diff
@@ -1,11 +1,11 @@
 import { AbstractParser } from '../../plugin-parser/src/AbstractParser';
-import { DomParsingEngine } from '../../plugin-dom/src/DomParsingEngine';
+import { XmlParsingEngine } from '../../plugin-xml/src/XmlParsingEngine';
 import { BlockquoteNode } from './BlockquoteNode';
 import { nodeName } from '../../utils/src/utils';
 
 export class BlockquoteDomParser extends AbstractParser<Node> {
-    static id = 'dom';
-    engine: DomParsingEngine;
+    static id = 'xml';
+    engine: XmlParsingEngine;
 
     predicate = (item: Node): boolean => {
```

Comment:
If they are supposed to parse `xml` then they should be an `AbstractParser<string>` I guess, and the predicate should take a `string` rather than a `Node`. Same with `parse`, if it's an "xml parser" then it should parse XML, that is `string`, not `Element`.